addons/spectator/functions/fn_drawTags.sqf

```diff
@@ -57,20 +49,16 @@ _unitTagSize = 1 * GVAR(unitTagScale);
     if(_veh != _x) then {
         _x = _veh;
         // is not unique.
-        if ((_doneVehicles pushBackUnique _x) == -1) then {
-            _render = false;
-        };
+        _render = if ((_doneVehicles pushBackUnique _x) == -1) then { false };
```

Comment:
If the condition is false this will nil value the _render variable due to lack of an else clause.
An alternative solution is to include the else clause
`_render = if ((_doneVehicles pushBackUnique _x) == -1) then { false } else { _render };`
However I believe this would be a worse solution than what was previously there (only changing the value of _render if the condition is true). I tested this in the debug console myself
```cpp
_test = "1";
_test = if (1==2) then { "2" };

test = isNil "_test";
/* test returns true */
```


This pattern has been used many times throughout.